runasadministrator_windows_test.go

```diff
@@ -85,11 +85,12 @@ func TestRunAsAdministratorMissingOSGroup(t *testing.T) {
 	_ = submitAndAssert(t, td, payload, "exception", "malformed-payload")
 }
 
+// TODO can we parameterize these to test both openpgp + ed25519 paths?
```

Comment:
I think we can replace this `TODO` with a comment that the payload isn't actually executed, because we should always get `malformed-payload` task resolution. Therefore, the `type` command should never actually run, so it doesn't matter too much what it does. The reason I put the `type` command in is in case it really does run, and then we see if it can actually log the private key file - then we know we're really in trouble! 😟